pkgs/development/python-modules/psutil/default.nix

```diff
@@ -16,7 +16,7 @@ buildPythonPackage rec {
 
   # arch doesn't report frequency is the same way
   # tests segfaults on darwin https://github.com/giampaolo/psutil/issues/1715
-  doCheck = stdenv.isDarwin || stdenv.isx86_64;
+  doCheck = if stdenv.isDarwin then false else stdenv.isx86_64;
```

Comment:
Why disable tests?

Reply:
For psutil tests don't work on darwin see: https://github.com/NixOS/nixpkgs/pull/82524